src/scanner.l

```diff
@@ -6901,6 +6908,31 @@ NONLopt [^\n]*
                                           yyextra->nestedComment=0;
                                           BEGIN(DocCopyBlock);
                                         }
+<DocBlock>{CMD}"ifile"{B}+"\""[^\n\"]+"\"" {
```

Comment:
Do we need to take care of escaped `\ifile` and `\iline` commands as well? I don't expect all cases can be handled by the rule on line 6889 as the match here is potentially longer.

Reply:
No I don't think so as there is an explicit rule for `{CMD}{CMD}` (around 22 lines upwards):
```
<DocBlock>({CMD}{CMD}){ID}/[^a-z_A-Z0-9] { // escaped command
                                          yyextra->docBlock << yytext;
                                        }

```